### PR TITLE
added proj description

### PR DESCRIPTION
**Pull request by PhilLar**
Time to describe the proj!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+`Short` is the tool to shorten URL.
+
+Installation
+------------
+
+Install `short` from sources, by running:
+
+```sh
+git clone https://github.com/PhilLar/short.git
```
Comment (souryogurt) [inline README.md]: Hey! You need to fix repo name to `https://github.com/PhilLar/shorten.git`.

File: README.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+-----
+You can shorten the URL:
+```sh
+short "https://www.reddit.com/r/ProgrammerHumor/comments/922bys/there_are_two_ways_to_write_errorfree_programs/"
+https://cleanuri.com/xyZM0Q
+```
+
+Contribute
+----------
+- Issue Tracker: https://github.com/PhilLar/short/issues
```
Comment (souryogurt) [inline README.md]: And here

File: README.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+- Source Code: https://github.com/PhilLar/short
```
Comment (souryogurt) [inline README.md]: And here too

**Review by souryogurt — CHANGES_REQUESTED** [review]
There are wrong repo name

File: README.md (hunk 3 above)
Comment (PhilLar) [reply README.md]: all fixed now

**Pull request merged**